vlm/src/main/scala/geotrellis/contrib/vlm/geotiff/GeoTiffReprojectRasterSource.scala

```diff
@@ -98,8 +98,7 @@ case class GeoTiffReprojectRasterSource(
         extent = gridExtent.extentFor(targetPixelBounds, clamp = true),
         cols = targetPixelBounds.width.toInt,
         rows = targetPixelBounds.height.toInt)
-      val sourceExtent =
-        closestTiffOverview.synchronized { targetRasterExtent.extent.reprojectAsPolygon(backTransform, 0.001).envelope }
+      val sourceExtent = targetRasterExtent.extent.reprojectAsPolygon(backTransform, 0.001).envelope
```

Comment:
Why did you remove `synchronized`? It creates stacktraces related to extent reprojection, check out https://github.com/geotrellis/geotrellis-contrib/pull/206#pullrequestreview-260115791